recipes/implot/all/CMakeLists.txt

```diff
@@ -0,0 +1,27 @@
+cmake_minimum_required(VERSION 3.4)
+project(implot CXX)
+set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+
+include(conanbuildinfo.cmake)
+conan_basic_setup()
+
+set(CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS ON)
+
+set(SOURCE_DIR source_subfolder)
+file(GLOB SOURCE_FILES ${SOURCE_DIR}/*.cpp)
+file(GLOB HEADER_FILES ${SOURCE_DIR}/*.h)
+
+add_library(${PROJECT_NAME} ${SOURCE_FILES})
+target_include_directories(${PROJECT_NAME} PRIVATE ${SOURCE_DIR})
+target_link_libraries(${PROJECT_NAME} PUBLIC ${CONAN_LIBS})
+
+include(GNUInstallDirs)
+
+install(TARGETS ${PROJECT_NAME}
+        RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
+        LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
+        ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR})
+install(FILES ${HEADER_FILES}
```

Comment:
this glob installed an `internal.h` files 😕

Reply:
Could you please elaborate a bit?
In my local .conan/data, there is only `implot.h` and `implot_internal.h` in the include directory.

Reply:
The naming is a little weird, but `*_internal.h` files are also a part of the api. They just mean "these stuff are unstable and may change from (minor) version to version". Similar to the demo file, the `imgui` recipe also packages the imgui_internal.h file.

Reply:
Evidence: https://github.com/epezent/implot/blob/master/implot_demo.cpp#L1956-L1962
```
// WARNING:
//
// You can use "implot_internal.h" to build custom plotting fuctions or extend ImPlot.
// However, note that forward compatibility of this file is not guaranteed and the
// internal API is subject to change. At some point we hope to bring more of this
// into the public API and expose the necessary building blocks to fully support
// custom plotters. For now, proceed at your own risk!
```